autokeras/blocks/basic.py

```diff
@@ -91,7 +92,13 @@ def __init__(
             int,
         )
         self.use_batchnorm = use_batchnorm
-        self.dropout = dropout
+        self.dropout = utils.get_hyperparameter(
+            dropout,
+            hyperparameters.Choice(
+                "dropout", [0.0, 0.2, 0.3, 0.4, 0.5, 0.6], default=0.0
```

Comment:
We prefer to keep the original search space of [0.0, 0.25, 0.5].
More value would add burden to the search algorithm.

Reply:
I reverted back to original search space